h/static/scripts/site.js

```diff
@@ -23,7 +22,7 @@ document.addEventListener('DOMContentLoaded', function () {
   // setup route
   var route = document.location.pathname;
   if (route.match('^/(new-homepage)?$')) {
```

Comment:
Could you remove the `new-homepage` option from this. Happy for that to be a separate commit/PR.